pyanaconda/modules/users/users.py

```diff
@@ -87,6 +87,8 @@ def process_kickstart(self, data):
         if data.rootpw.seen:
             self.set_can_change_root_password(False)
             self._rootpw_seen = True
+            if data.rootpw.allow_ssh:
+                self.set_root_password_ssh_login_allowed(True)
```

Comment:
I think we should always set this property if we get a kickstart. Please, call the following code outside of the `if` branch: `self.set_root_password_ssh_login_allowed(data.rootpw.allow_ssh)`

Reply:
I didn't comment because I thought the value can be also `None`, but given how it's parsed, the above is definitely better.

Reply:
Done